client/elements/emoji-selector.html

```diff
@@ -1,3 +1,5 @@
+<base href="https://polygit2.appspot.com/emojilib+muan+1.0.1/components/">
```

Comment:
I was nervous because I didn't load the actual demo with the server and everything, so i didn't want to break it. But if you think it'll be fine.... :)